### PR TITLE
[Adjustment][Vince] recolor dropdown navbar

### PR DESCRIPTION
**Pull request by VinceRaAle**
https://www.loom.com/share/305e4a22199a4ec9babb87571de352f1

### CODE REVIEW

File: components/Dropdown.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ function Dropdown(props: PropsInterface) {
 
   return (
     <div
-      className={classNames("absolute top-10 p-2 bg-white", {
+      className={classNames("absolute top-10 p-2 bg-[#2C317A]", {
```
Comment (ezerssss) [inline components/Dropdown.tsx]: add this color to `tailwind.config.js`, name it appropriately like `tmi-dark-blue` or smth

File: components/Header.tsx [hunk 2]
```diff
@@ -83,21 +83,15 @@ function Header() {
...
-                <Link
-                  className="p-2 hover:bg-tmi-hover-gray"
-                  href="/about#mission"
-                >
+                <Link className="p-2 hover:bg-[#242865]" href="/about#mission">
                   Mission
                 </Link>
                 <Link
-                  className="p-2 hover:bg-tmi-hover-gray"
+                  className="p-2 hover:bg-[#242865]"
```
Comment (ezerssss) [inline components/Header.tsx]: also put it in the tailwind config

File: components/Dropdown.tsx (hunk 1 above)
Comment (VinceRaAle) [reply components/Dropdown.tsx]: resolved

File: components/Header.tsx (hunk 2 above)
Comment (VinceRaAle) [reply components/Header.tsx]: resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**